egs2/spring_speech/local/data.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+SECONDS=0
+
+lang=
+release_name="SPRING_INX"
```

Comment:
Why not directly use SPRING_INX in the following context?

Reply:
The name of the file being downloaded will change depending on the lang argument in run.sh and the versions maintained in our back-end. We are using release_name for relative names of the files.